### PR TITLE
FLIP-2244: Update fl-wrapper to take in and apply aggregator and weights

### PR DESCRIPTION
**Pull request by MichaelOSullivanAnswer**
The following snapshots are of the `config.json` supplied to and the `config_fed_server.json` generated by the successful run that is also documented in the PR for FLIP-2243:

image

image

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -283,7 +283,9 @@ def upload_app(application_path: str):
             application_path,
             body["global_rounds"],
             body["trusts"],
-            body["ignore_result_error"] if "ignore_result_error" in body else False
+            body["ignore_result_error"] if "ignore_result_error" in body else False,
+            body["aggregator"],
```
Comment (handyandypandyandy) [inline app.py:287]: lets make sure these two have default values as well like the one above

Comment (MichaelOSullivanAnswer) [reply app.py:287]: or None?

Comment (handyandypandyandy) [reply app.py:287]: empty dict i think

**Review by handyandypandyandy — DISMISSED** [review]

File: utils/prepare_config.py [hunk 2]
```diff
@@ -58,6 +65,8 @@ def configure_server(app: str, global_rounds: int, trusts: List[str], ignore_res
         server_config["min_clients"] = len(trusts)
         server_config["workflows"][2]["args"]["participating_clients"] = trusts
         server_config["workflows"][1]["args"]["ignore_result_error"] = ignore_result_error
+        server_config["components"][2]["path"] = f'nvflare.app_common.aggregators.intime_accumulate_model_aggregator.{aggregator}'
```
Comment (handyandypandyandy) [inline utils/prepare_config.py:68]: I've just realised for this to work best, I think we're gonna have to make sure we specify the nvflare component name instead.
```suggestion
        server_config["components"][2]["name"] = aggregator
```
You'll also need to remove the "path" property from the component config in the base app as well https://github.com/AI4VBH/FLIP-fl-base-application/blob/develop/src/config/config_fed_server.json

NVFlare can use either the "name" or "path" property to reference components but "name" is only for stuff built into nvflare itself

**Comment by MichaelOSullivanAnswer** [comment]
The following snapshots are of the `config.json` supplied to and the `config_fed_server.json` generated by the successful run that is also documented in the PR for FLIP-2243:

image

image

**Review by mbaltrimas — APPROVED** [review]

**Review by handyandypandyandy — APPROVED** [review]

**Pull request merged**